docs/contents/components/data-display/tag/index.mdx

```diff
@@ -61,237 +70,120 @@ Here is an example of a `Tag` that includes an icon.
-    <Tag variant="outline" colorScheme="secondary">
-      Secondary
-    </Tag>
-    <Tag variant="outline" colorScheme="success">
-      Success
-    </Tag>
-    <Tag variant="outline" colorScheme="warning">
-      Warning
-    </Tag>
-    <Tag variant="outline" colorScheme="danger">
-      Danger
-    </Tag>
-    <Tag variant="outline" colorScheme="link">
-      Link
-    </Tag>
-    <Tag variant="outline" colorScheme="gray">
-      Gray
-    </Tag>
-    <Tag variant="outline" colorScheme="neutral">
-      Neutral
-    </Tag>
-    <Tag variant="outline" colorScheme="red">
-      Red
-    </Tag>
-    <Tag variant="outline" colorScheme="rose">
-      Rose
-    </Tag>
-    <Tag variant="outline" colorScheme="pink">
-      Pink
-    </Tag>
-    <Tag variant="outline" colorScheme="orange">
-      Orange
-    </Tag>
-    <Tag variant="outline" colorScheme="amber">
-      Amber
-    </Tag>
-    <Tag variant="outline" colorScheme="yellow">
-      Yellow
-    </Tag>
-    <Tag variant="outline" colorScheme="lime">
-      Lime
-    </Tag>
-    <Tag variant="outline" colorScheme="green">
-      Green
-    </Tag>
-    <Tag variant="outline" colorScheme="emerald">
-      Emerald
-    </Tag>
-    <Tag variant="outline" colorScheme="teal">
-      Teal
-    </Tag>
-    <Tag variant="outline" colorScheme="cyan">
-      Cyan
-    </Tag>
-    <Tag variant="outline" colorScheme="sky">
-      Sky
-    </Tag>
-    <Tag variant="outline" colorScheme="blue">
-      Blue
-    </Tag>
-    <Tag variant="outline" colorScheme="indigo">
-      Indigo
-    </Tag>
-    <Tag variant="outline" colorScheme="violet">
-      Violet
-    </Tag>
-    <Tag variant="outline" colorScheme="purple">
-      Purple
-    </Tag>
-    <Tag variant="outline" colorScheme="fuchsia">
-      Fuchsia
-    </Tag>
+    <For
+      each={[
+        { colorScheme: "primary" },
+        { colorScheme: "secondary" },
+        { colorScheme: "success" },
+        { colorScheme: "warning" },
+        { colorScheme: "danger" },
+        { colorScheme: "link" },
+        { colorScheme: "gray" },
+        { colorScheme: "neutral" },
+        { colorScheme: "red" },
+        { colorScheme: "rose" },
+        { colorScheme: "pink" },
+        { colorScheme: "orange" },
+        { colorScheme: "amber" },
+        { colorScheme: "yellow" },
+        { colorScheme: "lime" },
+        { colorScheme: "green" },
+        { colorScheme: "emerald" },
+        { colorScheme: "teal" },
+        { colorScheme: "cyan" },
+        { colorScheme: "sky" },
+        { colorScheme: "blue" },
+        { colorScheme: "indigo" },
+        { colorScheme: "violet" },
+        { colorScheme: "purple" },
+        { colorScheme: "fuchsia" },
+      ]}
+    >
+      {({ variant, colorScheme }, index) => (
+        <Tag key={index} variant="outline" colorScheme={colorScheme}>
+          {colorScheme[0].toUpperCase() +
+            colorScheme.slice(1, colorScheme.length)}
+        </Tag>
+      )}
+    </For>
   </Wrap>
 </VStack>
 ```
```

Comment:
```tsx
<VStack>
  <For each={["subtle","solid","outline"]}>
    {(variant,index)=>(
      <Wrap key={index} gap="md">
        <For 
          each={[
            "primary",
            "secondary",
            "success",
            "warning",
            "danger",
            "link",
            "gray",
            "neutral",
            "red",
            "rose",
            "pink",
            "orange",
            "amber",
            "yellow",
            "lime",
            "green",
            "emerald",
            "teal",
            "cyan",
            "sky",
            "blue",
            "indigo",
            "violet",
            "purple",
            "fuchsia",
          ]}
        >
          {(colorScheme,index)=>(
            <Tag key={index} variant={variant} colorScheme={colorScheme}>
              {toTitleCase(colorScheme)}
            </Tag>
          )}
        </For>
      </Wrap>
    )}
  </For>
</VStack>
```

Reply:
I think this is the most compact, but I'll leave it to you to decide how far you want to go.
```tsx
<VStack>
  <For each={["subtle","solid","outline"]}>
    {(variant,index)=>(
      <Wrap key={index} gap="md">
        <For each={[...SEMANTIC_COLOR_SCHEMES, ...COLOR_SCHEMES]}>
          {(colorScheme,index)=>(
            <Tag key={index} variant={variant} colorScheme={colorScheme}>
              {toTitleCase(colorScheme)}
            </Tag>
          )}
        </For>
      </Wrap>
    )}
  </For>
</VStack>
```

Reply:
@kaehehehe 
It does not seem to have been corrected.

Reply:
@108yen 
Are you referring to the constants `SEMANTIC_COLOR_SCHEMES` and `COLOR_SCHEMES`? Looking at [this PR](https://github.com/yamada-ui/yamada-ui/pull/3834), it seems that not using constants is the correct approach, so I intentionally did not make any changes."

Reply:
@kaehehehe 
I agree that.
Wouldn't you make it a double loop?
https://github.com/yamada-ui/yamada-ui/pull/3841#discussion_r1875655351

Reply:
@108yen 
good idea! I've implemented it.